jwql/jwql_monitors/generate_preview_images.py

```diff
@@ -798,7 +860,185 @@ def process_program(program, overwrite):
 
         # Create the nominal preview image and thumbnail
         try:
-            im = PreviewImage(filename, "SCI")
+            im = preview_img_from_file(filename, parsed)
+            if im is not None:
+                new_preview_counter += 1
+                thumbnail_files.extend(img.thumbnail_images)
+
+
+            #im = PreviewImage(filename, "SCI")  - level 3 files might not have sci extension
+
+
+            """
```

Comment:
I think all of this information is great. Do you think we should create a little readme in the `jwql_monitors` directory for the level 3 products and their descriptions there so we don't take up so many lines and break up the code?